src/java/fr/paris/lutece/portal/service/database/DAOUtilTransactionManager.java

```diff
@@ -60,7 +60,7 @@
 public class DAOUtilTransactionManager extends DataSourceTransactionManager implements PluginEventListener
 {
     private static final long serialVersionUID = -654531540978261621L;
-    private Logger _logger = Logger.getLogger( "lutece.debug.sql.tx" );
+    private transient Logger _logger = Logger.getLogger( "lutece.debug.sql.tx" );

```

Comment:
I don't think this correctly makes the class serializable. On reactivation, the field `_logger` will be `null`, and every use will throw. You should add a getter for the logger, and access it through it

Reply:
Will be corrected